certora/helpers/CallHookHelper.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+contract CallHookHelper {
+    // This funciton is just a way to save msg.sender and msg.value
+    // from unresolved calls. This gets routed to a CVL function.
+    function HarnessCallHook() external {}
```

Comment:
This should be payable, otherwise every successful call to it would force msg.value to be zero.

Reply:
Perhaps the summary of HarnessCallHook doesn't take into account its msg.value restriction.

Reply:
I deleted this file now. (There is no summary to HarnessCallHook and it is just not used).